### DISCUSSION TITLE
Dawn copy theme approved on themeforest by a TF author.

### DISCUSSION THREAD

**Discussion by mohinht** [Shopify/dawn#2089] [General]
Hello,
TF link: https://themeforest.net/item/atoms-clothing-accessories-shopify-theme/40471921
Dawn copy theme: https://joomlart-atoms.myshopify.com/
Password: joom

I feel very sad as a Developer. How an author Shopify Free theme approved on Themeforest marketplace for selling without changing anything. Shopify provides a starter theme to develop a theme.  But not sell the dawn copy version. Shopify should take an action against the author.

https://user-images.githubusercontent.com/73767253/199417991-ae7a4a77-0180-49c2-96a7-c699008d6208.mp4

Thank you!

**Comment by karinstorm** [comment]
Actually, when I review the Dawn copy theme: https://joomlart-atoms.myshopify.com/ I see a important feature that is not available in the original Dawn theme, that is the option to switch from grid view to list view on a collections page: https://joomlart-atoms.myshopify.com/collections/furniture
